manifests/e/EdrawSoft/EdrawMax/CN/12.0.7.966/EdrawSoft.EdrawMax.CN.locale.en-US.yaml

```diff
@@ -0,0 +1,62 @@
+# Created with YamlCreate.ps1 v2.2.0 using InputObject 🤖 $debug=QUSU.CRLF..7-2-9
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: EdrawSoft.EdrawMax.CN
+PackageVersion: 12.0.7.966
+PackageLocale: en-US
+Publisher: EdrawSoft Co.,Ltd.
+PublisherUrl: https://www.edrawsoft.cn/
+PublisherSupportUrl: https://www.edrawmax.cn/helpcenter/pc/home
+PrivacyUrl: https://www.edrawsoft.cn/private-policy/
+Author: SHENZHEN EDRAW SOFTWARE CO.,LTD.
+PackageName: EdrawMax
+PackageUrl: https://www.edrawsoft.cn/edrawmax/
+License: Proprietary
+LicenseUrl: https://www.edrawsoft.cn/edrawmax-term-services/
+Copyright: Copyright © 2023 EdrawSoft. All rights reserved.
+# CopyrightUrl:
+ShortDescription: One-stop office diagram tool
+Description: EdrawMax is a design software that integrates office diagramming, engineering diagramming, image-text editing and flyer design, with which you can create all kinds of diagrams easily without any drawing skills.
+# Moniker:
+Tags:
+- chart
+- design
+- diagram
+- draw
+- gantt
+- graph
+- graphics
+- mind-map
+- mind-mapping
+- mindmap
+- uml
+# Agreements:
+ReleaseNotes: |-
+  【新增】泳道图冻结窗格功能
+  当泳道较长时，使用冻结窗格功能可以锁定泳道图标题行/列，使其他部分在滚动时可见
+
+  【新增】时间轴支持调节每个里程碑的间距
+  当鼠标浮动到时间轴上的里程碑圆点上时，圆点变大即代表可以自由拖动圆点来调整每个里程碑的间距
+
+  【新增】插入超链接后
+  展开超链接菜单后支持显示链接地址
+  幻灯片视图下支持超链接跳转
+
+  【其他】
+  根据用户反馈修复了插入注释的字体与系统设置的字体不一致问题
+  根据用户反馈解决了调整文本框时，文本框与文字自适应的问题
+  根据用户反馈优化了制图时浮动工具栏遮住形状的问题
+  根据用户反馈修复了组织架构图的布局预览图与实际展现不匹配的问题
+  根据用户反馈修复了甘特图今日红线显示异常的问题
```

Comment:
```suggestion
  [New] Swimming Lane map freeze pane function
  When swimlanes are long, use the Freeze Pane feature to lock the swimlane graph header row/column, so that other sections are visible while scrolling.

  [New] The time axis supports adjusting the spacing of each milestone.
  When the mouse hovers over the milestone dots on the timeline, the dots will become larger, which means that you can freely drag the dots to adjust the distance between each milestone.

  [New] After inserting a hyperlink:
  Supports display link address after expanding the hyperlink menu.
  Supports hyperlink jump in slide view.

  [other]
  Based on user feedback - Fixed the issue that the font for inserting comments was inconsistent with the font set by the system.
  Based on user feedback - Solved the problem of adapting the text box and text when adjusting the text box.
  Based on user feedback - Optimized the problem that the floating toolbar covered the shape when drawing.
  Based on user feedback - Fixed the problem that the layout preview of the organization chart did not match the actual display.
```